2012-04-29  Aurelien Jarno  <aurelien@aurel32.net>

	* locales/de_AT: Change month name from "Feber" to "Februar". 

---
 localedata/locales/de_AT |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/localedata/locales/de_AT
+++ b/localedata/locales/de_AT
@@ -102,7 +102,7 @@
         "<U0053><U0065><U0070>";"<U004F><U006B><U0074>";/
         "<U004E><U006F><U0076>";"<U0044><U0065><U007A>"
 mon     "<U004A><U00E4><U006E><U006E><U0065><U0072>";/
-        "<U0046><U0065><U0062><U0065><U0072>";/
+        "<U0046><U0065><U0062><U0072><U0075><U0041><U0072>";/
         "<U004D><U00E4><U0072><U007A>";/
         "<U0041><U0070><U0072><U0069><U006C>";/
         "<U004D><U0061><U0069>";/
